aca_entities.gemspec

```diff
@@ -33,15 +33,16 @@ Gem::Specification.new do |spec|
   spec.executables = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
   spec.add_dependency 'deep_merge'
-  spec.add_dependency 'dry-monads', '~> 1.2'
-  spec.add_dependency 'dry-struct', '~> 1.0'
-  spec.add_dependency 'dry-types', '~> 1.0'
```

Comment:
@TristanB17 @adamrunner Why are we bumping the versions of these gems?

Reply:
To get it working with the latest versions of the gems that are compatible? We can roll back to older versions if needed, but these updated versions appear to work so they will reduce overhead in doing later version upgrades.

Reply:
@adamrunner We are trying to understand if these are changes that you must need to make FTI work.